### ISSUE TITLE
[Question/Feature Request] Does reading mode wait for browser to render pages first?

### ISSUE THREAD

**Issue by nhantrn** [ankit/stylebot#614]
If so my request is to intercept the DOM and parse out the  scraps before sending it the browser for render. This way it reduce the browser rendering job and remove all the js garbage in all one fell blow, as the saying goes.

**Comment by ankit** [comment]
@nhantrn yes the readability mode can be optimized for faster rendering but it will be non-trivial since it will work differently from how the custom CSS is applied. Marking this as a feature request, thanks
